src/plugins/expressions/common/util/create_error.ts

```diff
@@ -21,7 +21,9 @@ import { ExpressionValueError } from '../../common';
 
 type ErrorLike = Partial<Pick<Error, 'name' | 'message' | 'stack'>>;
 
-export const createError = (err: string | Error | ErrorLike): ExpressionValueError => ({
+export const createError = (
+  err: string | Error | ErrorLike | ExpressionValueError['error']
```

Comment:
Doesn't `ExpressionValueError['error']` fall into the `ErrorLike` category?
And if not, why? 
:thinking:

Reply:
Yes it’s because of the original property - no strong opinion on this, I can change it to & { orginial?: Error } if you want